cdp-agentkit-core/typescript/docs/media/CONTRIBUTING.md

```diff
@@ -1,40 +1,63 @@
 # CDP Agentkit Contributing Guide
-
 Thank you for your interest in contributing to CDP Agentkit! We welcome all contributions, no matter how big or small. Some of the ways you can contribute include:
 - Adding new actions to the core package
-- Creating new AI framework extensions
+- Updating existing Langchain Toolkits or adding new Langchain Toolkits to support new tools
+- Creating new AI frameworks extensions
 - Adding tests and improving documentation
 
-## Development
-
-### Prerequisites
-- Node.js 18 or higher
-- npm for package management
-
 ### Set-up
 
 Clone the repo by running:
 
 ```bash
-git clone git@github.com:coinbase/cdp-agentkit-nodejs.git
-cd cdp-agentkit-nodejs
+git clone git@github.com:coinbase/cdp-agentkit.git
 ```
 
+## Python Development
+### Prerequisites
+- Python 3.10 or higher
+- Rust/Cargo installed ([Rust Installation Instructions](https://doc.rust-lang.org/cargo/getting-started/installation.html))
+- Poetry for package management and tooling
+  - [Poetry Installation Instructions](https://python-poetry.org/docs/#installation)
+
+`cdp-langchain` also requires a [CDP API Key](https://portal.cdp.coinbase.com/access/api).
+
+### Development Tools
+#### Formatting
+`make format`
+
+#### Linting
+- Check linter
```

Comment:
Do we want Python instructions to be shipped in the Typescript API Reference Docs `CONTRIBUTING.md`?

Reply:
probably not? I just followed the precedence set in the twitter-langchain docs, but will remove from both